presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/RpmController.java

```diff
@@ -197,4 +220,24 @@ public boolean isRunning()
     {
         return isInstalled() && executeCommand("service", "presto", "status") == 0;
     }
+
+    /* Utility methods for running commands */
+
+    private CommandResult executeCommandResult(String... commandArray)
+            throws PrestoManagerException
+    {
+        return executor.runCommand(commandArray);
+    }
+
+    private int executeLongCommand(String... commandArray)
+            throws PrestoManagerException
```

Comment:
I did.

This method is a convenience so you can write `executeLongCommand(...)` instead of `executor.runLongcommand` throughout the file.